### DISCUSSION TITLE
Help Request: Unable to log into admin dashboard

### DISCUSSION THREAD

**Discussion by yujioshiro** [TEAMMATES/teammates#12377] [Help requests] [answered]
### Environment

- **Commit SHA:** 2e616118df5cc205854e2d8b294a27e937e8df59
- **Operating System:** MacOS 12.6.4 (Intel)
- **JDK Version:** "11.0.18" 2023-01-17 LTS
- **Node.js Version:** v18.15.0
- **Chrome Version:** Version 112.0.5615.121 (Official Build) (x86_64)

### Intention

Attempting to log into the admin dashboard gives "You are not authorized to view the page" error (included browser console log if needed):
![](https://user-images.githubusercontent.com/19900334/232187426-919af71e-2bd6-4b83-9893-6578ee1689e4.gif)

docker command:
<img width="434" alt="image" src="https://user-images.githubusercontent.com/19900334/232259075-b3e8bd30-766f-4c2e-bb16-fbed959fe341.png">

frontend and backend servers:
<img width="547" alt="image" src="https://user-images.githubusercontent.com/19900334/232259092-14e1c637-fc1e-42ff-add3-db9857dfa0e8.png">

### Resolution Attempt

 - deleted, forked, and cloned the repo 3 times and followed the [setting up guide](https://teammates.github.io/teammates/setting-up.html) and [development guidelines](https://teammates.github.io/teammates/development.html)
 - changed one of the build.properties values as suggested [here](https://github.com/TEAMMATES/teammates/issues/12288#issuecomment-1509498950)
 - restarted my computer
 - tried logging on via Firefox and Safari
 - Here are things that were suggested to me in [issue #12288](https://github.com/TEAMMATES/teammates/issues/12288).

**Comment by jasonqiu212** [comment]
Hi @yujioshiro, Thanks for shifting over to the discussion forum here! 

Your setup of `build.properties` looks good to me! I suspect there might be some problems with the backend server:

1. When you try to access the site `http://localhost:4200/web/admin/home`, can I check if the terminal window running the `./gradlew serverRun` generates any errors? If so, you may want to try re-starting the docker containers and re-running `./gradlew serverRun`
2. Also, could you try clicking on `Login` then `Instructor Login` in the top right corner? Does the page redirect you to a login page?

**Reply by yujioshiro** [reply]
Hello @jasonqiu212, still learning about contributing to projects, sorry for throwing so many messages in the issue 😅.

1. Here is the frontend and backend server terminals. No new logs. I also show the docker console, no changes there either.
![admin-error](https://user-images.githubusercontent.com/19900334/232262364-73c97db5-0e16-417b-bb99-125fe3159f68.gif)


2. When I click on  `Login` and select `Instructor Login` it just refreshes and, again, no new logs in the terminals.
![login-error](https://user-images.githubusercontent.com/19900334/232262447-4ab6f546-065e-4138-bd80-85d7d6ac2a38.gif)

**Reply by jasonqiu212** [reply, marked as answer]
Hi no worries! we're all here to learn :)

I believe I see the issue. There is some problem when starting the backend server, as shown in the screenshot below:

<img width="799" alt="Screen Shot 2023-04-16 at 12 29 34 PM" src="https://user-images.githubusercontent.com/25262042/232266710-9f6a5dc8-beb0-403f-9874-ad01ac162968.png">

After some further digging on the [troubleshooting guide](https://teammates.github.io/teammates/troubleshooting-guide.html), I think your `FileNotFoundException` may be caused by the working directory containing a space inside (i.e. Your `Current Projects` folder in the directory).

As per the guide, this can be resolved by moving the repository to another directory.

**Reply by yujioshiro** [reply]
Oh my... I assumed everything started okay because it was at 85%. I will need to do dig deeper next time I run into issues. 

I am now seeing the correct login page. Thank you!!
<img width="531" alt="image" src="https://user-images.githubusercontent.com/19900334/232335197-a9cb3300-9a8f-4009-bca4-e695b6312ea0.png">
